crates/staged-sync/src/config.rs

```diff
@@ -153,6 +153,12 @@ mod tests {
     use super::Config;
     #[test]
     fn can_serde_config() {
+        // Delete config file if it exists, as it can be outdated and
+        // make the test fail. It will be then created.
+        let config_path = confy::get_configuration_file_path("test", None).unwrap();
```

Comment:
we should use tempdir for this instead since we are testing ... ser/deser ... i guess @onbjerg? let's make a test name more descriptive as well

Reply:
What else could the test be named?

Reply:
since it's only loading & deserializing, let's call it `can_load_and_deserialize_config`